generators/app/index.js

```diff
@@ -53,7 +55,6 @@ module.exports = class extends Generator {
             // Ask for extension type
             askForType: () => {
                 let extensionType = generator.options['extensionType'];
-
                 if (extensionType) {
```

Comment:
sorry for asking again. can you add comment on top of the lines you are changing for ADS? like "// {{SQL CARBON EDIT}}" similar to this: [https://github.com/microsoft/azuredatastudio/blob/ddb210e971906d307bad7875d6aa5445bd856f2f/src/vs/platform/product/common/productService.ts#L69](https://github.com/microsoft/azuredatastudio/blob/ddb210e971906d307bad7875d6aa5445bd856f2f/src/vs/platform/product/common/productService.ts#L69)